pkg/kv/kvserver/store_snapshot.go

```diff
@@ -239,7 +242,10 @@ func (msstw *multiSSTWriter) Close() {
 // 3. Two lock-table key ranges (optional)
 // 4. User key range
 func (kvSS *kvBatchSnapshotStrategy) Receive(
-	ctx context.Context, stream incomingSnapshotStream, header kvserverpb.SnapshotRequest_Header,
+	ctx context.Context,
+	stream incomingSnapshotStream,
+	header kvserverpb.SnapshotRequest_Header,
+	bytesRcvd *metric.Counter,
```

Comment:
nit: let's make `bytesRcvd` a field on `kvBatchSnapshotStrategy` instead.

Reply:
Heh, what do you think about Nathan's point here: https://github.com/cockroachdb/cockroach/commit/550ba32c8cd17b0b0fde07682fc6fbb6dee0322a?

Reply:
Hm, right. I agree with his change, because there's no point in having that field on the struct -- it's just a local count that's being returned. However, these metrics seem more like infrastructure, which I'd usually hook up on the struct rather that plumb them through calls, similarly to the rate limiter.

Not a bit deal though, whatever you prefer is fine with me.